### ISSUE TITLE
Shadowmourne last quest in the chain, a sealed chest turn in items

### ISSUE THREAD

**Issue by sowhat2** [dalaranwow/dalaran-wow#5873]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After i kill Lich King i colect an item named A Sealed Chest, containing bunch of items wich i turn in to Highlord Mograine, Jain, Silvanas, King Terenas, Muradin. I can turn the item to Highlord Mograine (i saw the quest above his head), but for the rest of them i couldn't see any quest, and could turn in the items).

**Expected behaviour**: Tell us what should happen instead
I should be able to see a quest mark above everybody's heads like in the case of Highlord Mograine so i can turn in the items from the sealed chest.
**Steps to reproduce the problem**: n/a

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by sowhat2** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/5872#issuecomment-609420881
Golganeth i can't abandon any quests because i do not have any, i only have the items that i need to turn in to each of the NPC's i mentioned in the issue. The problem is that they do not generate a quest.
